integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -825,16 +825,20 @@ void testHiveCatalogCreatedByGravitino() throws InterruptedException {
                     "thrift://%s:%s",
                     containerSuite.getHiveContainer().getContainerIpAddress(),
                     HiveContainer.HIVE_METASTORE_PORT))
-            .put("hive.immutable-partitions", "true")
-            .put("hive.target-max-file-size", "1GB")
-            .put("hive.create-empty-bucket-files", "true")
-            .put("hive.validate-bucketing", "true")
+            .put("trino.bypass.hive.immutable-partitions", "true")
+            .put("trino.bypass.hive.target-max-file-size", "1GB")
+            .put("trino.bypass.hive.create-empty-bucket-files", "true")
+            .put("trino.bypass.hive.validate-bucketing", "true")
             .build());
     Catalog catalog = createdMetalake.loadCatalog(NameIdentifier.of(metalakeName, catalogName));
-    Assertions.assertEquals("true", catalog.properties().get("hive.immutable-partitions"));
-    Assertions.assertEquals("1GB", catalog.properties().get("hive.target-max-file-size"));
-    Assertions.assertEquals("true", catalog.properties().get("hive.create-empty-bucket-files"));
-    Assertions.assertEquals("true", catalog.properties().get("hive.validate-bucketing"));
+    Assertions.assertEquals(
+        "true", catalog.properties().get("trino.bypass.hive.immutable-partitions"));
```

Comment:
Does the key still start with `trino.bypass` when we load the catalog from Gravitino?

Reply:
We pass the configuration that starts with trino.bypass to the original Trino connector.